README.md

```diff
@@ -88,6 +88,7 @@ Showcase for projects built during [#30DaysOfAppwrite](https://30days.appwrite.i
 * [Flutter Appwrite Todo](https://github.com/devkishor8007/todo-with-flutter/tree/todo_appwrite/flutter_appwrite)
 * [KnowYourFood](https://github.com/LeBraveLittleToaster/KnowYourFood)
 * [Budgetist](https://github.com/intel-raja/budgetist_kakeibo#budgetist_kakeibo)
+* [Expo Appwrite Store your Medicines](https://github.com/Neeleshrj/store-your-meds.git)
```

Comment:
I don't think this should be part of the 30 days of Appwrite challenge. Would you please move it to the React section?